bigbluebutton-html5/imports/ui/services/features/index.js

```diff
@@ -85,5 +85,5 @@ export function isReactionsEnabled() {
 }
 
 export function isTimerFeatureEnabled() {
-  return getDisabledFeatures().indexOf('timer') === -1;
+  return getDisabledFeatures().indexOf('timer') === -1 && Meteor.settings.public.timer;
```

Comment:
```suggestion
  return getDisabledFeatures().indexOf('timer') === -1 && Meteor.settings.public.timer.enabled;
```
Also, consider changing other places in the code where `Meteor.settings.public.timer` is being used to use this `isTimerFeatureEnabled` function.